.baseline/findbugs/excludeFilter.xml

```diff
@@ -1,5 +1,9 @@
 <!-- See http://findbugs.sourceforge.net/manual/filter.html for syntax -->
 <FindBugsFilter>
+    <!-- ignore generated code -->
+    <Match>
+        <Class name="~.*\.*Immutable.*" />
```

Comment:
this project does have an outdated processors plugin, we should bump the version to 0.2.4 (latest)